<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingCart</class>
 <widget class="QDialog" name="ShoppingCart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkoutButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
